src_python/habitat_sim/simulator.py

```diff
@@ -248,7 +248,11 @@ def _config_pathfinder(self, config: Configuration) -> None:
                 f"Recomputing navmesh for agent's height {default_agent_config.height} and radius"
                 f" {default_agent_config.radius}."
             )
-            self.recompute_navmesh(self.pathfinder, needed_settings)
+            self.recompute_navmesh(
+                self.pathfinder,
+                needed_settings,
```

Comment:
So weird that this isn't in the navmesh setting struct...

Reply:
Actually, don't we need to change the if statement above this to always recompute the navmesh if include_static_objects=True, since we can't guarantee that the navmesh setting caching will work in that case? This code will not be run currently if a navmesh computed with include_static_objects=False is cached as part of the dataset, right?